edx/frontend-app-admin-portal
# ENT-2953: Connect Assign Endpoint for subscription page

- When assigning licenses in the admin dash subscriptions page, the expected data is sent to the license manager service via API call.
- The “Assign Learner(s)” button has a spinner while the API call is pending, and the modal closes after successful assignment 
- When the assign endpoint errors, an error alert is shown in the “Add Users” modal 


**TESTING INSTRUCTIONS:**

- Make sure you have a SubscriptionPlan
- To add a subscription plan visit: http://localhost:18170/admin/subscriptions/subscriptionplan/add/
- Make sure your plan has available (unassigned) licenses associated with it
- Go to http://localhost:1991/{enterprise_uuid}/admin/subscriptions and click on Add Users, enter an email and then assign
- Verify that all three points written above are working fine





**Note**: For testing, you will have to hardcode the subscription_uuid value here for now: https://github.com/edx/frontend-app-admin-portal/blob/master/src/components/subscriptions/data/service.js#L75.
This will be fixed once @saleem’s PR for ENT-2952 is merged.
